### DISCUSSION TITLE
Regarding Modularization of Components

### DISCUSSION THREAD

**Discussion by psankhe28** [camicroscope/GSOC#21] [Q&A]
I want to discuss some implementation details about this project. Which would be the best place to connect with mentor and discuss this?

**Comment by nanli-emory** [comment]
Hi @psankhe28 sorry for the delay in responding. We are receiving a high volume of very general questions, and we are responding to project-specific questions only. Have you submitted your proposal? We want to see what you thought about the modularization of components for camicroscope.

**Comment by psankhe28** [comment]
Yeah, I have submitted it on 27th of March.
